### DISCUSSION TITLE
Frictionless Repository

### DISCUSSION THREAD

**Discussion by roll** [frictionlessdata/frictionlessdata.io#682] [Show and tell]
Hi **Frictionless Community**,

We want to share with you our new project called **Frictionless Repository**. It's a Github Action for continuous data validation. It's in a public beta stage so please share your feedback with us!

# [Frictionless Repository](https://repository.frictionlessdata.io/)

![fr](https://user-images.githubusercontent.com/557395/119805014-2ace1500-bee1-11eb-8718-6955c9ded9c4.png)

<hr >

## [Demo Project](https://github.com/frictionlessdata/repository-demo)

[![Frictionless](https://github.com/frictionlessdata/repository-demo/actions/workflows/frictionless.yaml/badge.svg)](https://repository.frictionlessdata.io/report?user=frictionlessdata&repo=repository-demo&flow=frictionless)
[![Experimental](https://github.com/frictionlessdata/repository-demo/actions/workflows/experimental.yaml/badge.svg)](https://repository.frictionlessdata.io/report?user=frictionlessdata&repo=repository-demo&flow=experimental)

![Example](https://user-images.githubusercontent.com/557395/119179704-8b87c880-ba6f-11eb-8222-6f5f61a1e98b.png)

**Comment by aborruso** [comment]
Great job and idea!

**Reply by roll** [reply]
Thanks, @aborruso! 

Wanted to ping you first but you're faster than me :+1: @lwinfree @sapetti9 Let's ping other people who might be interested in early adoption.
